controller/workitem.go

```diff
@@ -373,7 +376,21 @@ func ConvertWorkItem(request *goa.RequestData, wi *app.WorkItem, additional ...W
 				op.Attributes[workitem.SystemDescriptionRendered] =
 					rendering.RenderMarkupToHTML(html.EscapeString((*description).Content), (*description).Markup)
 			}
-
+		case workitem.SystemCodebase:
+			if val != nil {
+				op.Attributes[name] = val
+				cb := val.(codebase.Codebase)
+				// Following format is TBD
+				urlparams := fmt.Sprintf("/codebase/generate?repo=%s&branch=%s&file=%s&line=%d", cb.Repository, cb.Branch, cb.FileName, cb.LineNumber)
+				editCodebaseURL := rest.AbsoluteURL(request, url.QueryEscape(urlparams))
+				op.Relationships.Codebase = &app.RelationGeneric{
```

Comment:
@aslakknutsen thanks I will move it to links, but do you mean move that URL to links.meta["doit"] or to links.doit ?